app/assets/javascripts/discourse/app/components/post-text-selection-toolbar.gjs

```diff
@@ -33,25 +33,35 @@ export default class PostTextSelectionToolbar extends Component {
       {{this.appEventsListeners}}
     >
       <div class="buttons">
-        {{#if this.embedQuoteButton}}
-          <DButton
-            @icon="quote-left"
-            @label="post.quote_reply"
-            @title="post.quote_reply_shortcut"
-            class="btn-flat insert-quote"
-            @action={{@data.insertQuote}}
-          />
+        {{#if this.showPrimaryActions}}
+          {{#if this.embedQuoteButton}}
+            <DButton
+              @icon="quote-left"
+              @label="post.quote_reply"
+              @title="post.quote_reply_shortcut"
+              class="btn-flat insert-quote"
+              @action={{@data.insertQuote}}
+            />
+          {{/if}}
+
+          {{#if @data.canEditPost}}
+            <DButton
+              @icon="pencil-alt"
+              @label="post.quote_edit"
+              @title="post.quote_edit_shortcut"
+              class="btn-flat quote-edit-label"
+              {{on "click" this.toggleFastEdit}}
+            />
+          {{/if}}
         {{/if}}
 
-        {{#if @data.canEditPost}}
-          <DButton
-            @icon="pencil-alt"
-            @label="post.quote_edit"
-            @title="post.quote_edit_shortcut"
-            class="btn-flat quote-edit-label"
-            {{on "click" this.toggleFastEdit}}
-          />
-        {{/if}}
+        <PluginOutlet
+          @name="post-text-after-primary-actions"
+          @outletArgs={{hash
+            data=@data
+            hidePrimaryActions=this.hidePrimaryActions
```

Comment:
I am wondering if it would make sense to have a `showPrimaryActions` too or maybe just a `togglePrimaryActions`:

```javascript
  @action
  hidePrimaryActions(value) {
    this.showPrimaryActions = value;
  }
```

Reply:
Ah yes, thanks, this is a good idea I will update it to be a toggle